src/EnergyPlus/DataSizing.cc

```diff
@@ -1094,4 +1317,64 @@ Real64 OARequirementsData::calcOAFlowRate(EnergyPlusData &state,
     return OAVolumeFlowRate;
 }
 
+Sched::Schedule *OARequirementsData::getZoneFlowFracSched(EnergyPlusData &state, bool notAllSame)
```

Comment:
Some of the iaq methods apply the zone OA flow schedule to the OA requirements for CO2 or contaminants.